### PR TITLE
Fetchfriends

### LINKED ISSUE #102: Import Facebook Friends into Redux Store
When I log in, I should have my friends put into the redux store so that I can access that data in the app.

### PR DESCRIPTION
**Pull request by tiffeli**
1) friends is now added to redux state as:
[ { name: 'Alex Su', id: '123' },
  { name: 'Alex Bignell', id: '456' },
  { name: 'Jake Shasteen', id: '789' } ]
2) state.user object now using "id" as key instead of "uid", to be consistent. 
yay fraaandssss.

Closes #102

### CODE REVIEW

File: app/actions/action_user.js [hunk 1]
```diff
@@ -23,7 +29,7 @@ export const firebase_check = (userCredentials) => {
     checkIfUserExists(id, (userExist) => {
       if(!userExist) {
         let userInfo={};
-        userInfo.uid = id;
+        userInfo.id = id;
```
Comment (bignell) [inline app/actions/action_user.js]: :+1:

File: app/actions/action_user.js [hunk 2]
```diff
@@ -8,11 +8,17 @@ export const logIn = (payload) => {
...
 };
-
+export const updateFriends = (payload) => {
+  return{
+    type: UPDATE_FRIENDS,
+    payload
+  };
+};
 export const firebase_check = (userCredentials) => {
   let id = userCredentials.userId;
```
Comment (bignell) [inline app/actions/action_user.js]: The reassignment might be confusing, but with ES6 you can shorten this to:
`let { id: userId, token } = userCredentials`

**Comment by jake-shasteen** [comment]
This looks good to me. :heavy_check_mark: 
I also tested it in conjunction with #106 and it works as expected.
